### PR TITLE
pytorch_vision_resnext 번역

### PR DESCRIPTION
**Pull request by JunyongKang**
pytorch_vision_resnext.md 파일의 번역본입니다.

## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #41

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

### CODE REVIEW

File: pytorch_vision_resnext.md [hunk 1]
```diff
@@ -52,51 +50,51 @@ preprocess = transforms.Compose([
...
-# move the input and model to GPU for speed if available
+# gpu를 사용할 수 있다면, 속도를 위해 입력과 모델을 gpu로 옮김
 if torch.cuda.is_available():
     input_batch = input_batch.to('cuda')
     model.to('cuda')
 
 with torch.no_grad():
     output = model(input_batch)
-# Tensor of shape 1000, with confidence scores over Imagenet's 1000 classes
+# output은 shape가 [1000]인 Tensor 자료형이며, 이는 Imagenet 데이터셋의 각 클래스에 대한 모델의 확신도(confidence)를 나타냄.
```
Comment (yh00214) [inline pytorch_vision_resnext.md:62]: 1. Imagenet에 대한 지식이 부족한 분들이 본다고 생각한다면, 원문을 살려 'Imagenet 데이터셋의 1000개의 각 클래스들에 대한 확신도' 같이 1000개의 클래스를 명시해 주는 것이 좋을 것 같습니다.
2. 다른 번역 문장의 어미와 비교했을 때, 통일성을 위해 '나타냅니다.'로 바꾸는 것을 제안합니다.

**Review by yh00214 — CHANGES_REQUESTED** [review]
수고하셨습니다! 훌륭한 번역이었습니다.
중간에 한 군데만 코멘트를 달았으니 참고해주시기 바랍니다.

File: pytorch_vision_resnext.md (hunk 1 above)
Comment (FollowerOfScriabin) [reply pytorch_vision_resnext.md]: 리뷰사항 반영하였습니다. 감사합니다.

File: pytorch_vision_resnext.md [hunk 2]
```diff
@@ -52,51 +50,51 @@ preprocess = transforms.Compose([
...
 
-### Model Description
+### 모델 설명
 
-Resnext models were proposed in [Aggregated Residual Transformations for Deep Neural Networks](https://arxiv.org/abs/1611.05431).
-Here we have the 2 versions of resnet models, which contains 50, 101 layers repspectively.
-A comparison in model archetechure between resnet50 and resnext50 can be found in Table 1.
-Their 1-crop error rates on imagenet dataset with pretrained models are listed below.
+Resnext 모델은 논문 [Aggregated Residual Transformations for Deep Neural Networks]에서 제안되었습니다. (https://arxiv.org/abs/1611.05431).
+이중 두가지 버전의 모델 성능은 아래와 같습니다. 각 모델의 레이어 개수는 각 50, 101개입니다.
```
Comment (zhoonit) [inline pytorch_vision_resnext.md:89]: Here we have the 2 versions of resnet models, 의 의미를 살려 아래와 같이 번역하면 어떨지 제안드립니다 :)

```suggestion
여기서는 50개의 레이어와 101개의 레이어를 가지는 2개의 resnet 모델을 제공하고 있습니다.
각 모델 성능은 아래와 같습니다.
```

Comment (FollowerOfScriabin) [reply pytorch_vision_resnext.md]: 리뷰 사항 반영하였습니다. 감사합니다.

File: pytorch_vision_resnext.md [hunk 3]
```diff
@@ -25,23 +25,21 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'resnext50_32x4d', pretrained=T
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
-
-Here's a sample execution.
+모든 사전 훈련된 모델들은 입력 이미지가 동일한 방식으로 정규화되었다고 상정합니다.
```
Comment (Taeyoung96) [inline pytorch_vision_resnext.md:28]: `상정합니다`는 오타일까요?  
`가정합니다`라고 바꾸는 것이 조금 더 적절해 보입니다.

File: pytorch_vision_resnext.md [hunk 4]
```diff
@@ -25,23 +25,21 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'resnext50_32x4d', pretrained=T
...
+즉, 미니 배치(mini-batch)의 3-채널 RGB 이미지들은 `(3 x H x W)`의 형태를 가지며, 해당 `H`와 `W`는 최소 `224` 이상이어야 합니다.
+각 이미지는 `[0, 1]`의 범위 내에서 로드되어야 하며, `mean = [0.485, 0.456, 0.406]` 과 `std = [0.229, 0.224, 0.225]`을 이용해 정규화되어야 합니다.
```
Comment (Taeyoung96) [inline pytorch_vision_resnext.md:30]: `로드되어야`라는 단어도 `불러와야`로 번역하는 것이 어떤지 제안드립니다.

File: pytorch_vision_resnext.md [hunk 5]
```diff
@@ -52,51 +50,51 @@ preprocess = transforms.Compose([
     transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
 ])
 input_tensor = preprocess(input_image)
-input_batch = input_tensor.unsqueeze(0) # create a mini-batch as expected by the model
+input_batch = input_tensor.unsqueeze(0) # 모델에서 가정하는대로 미니배치 생성
```
Comment (Taeyoung96) [inline pytorch_vision_resnext.md:53]: `모델에서 가정하는 대로 미니 배치 생성`으로 띄어쓰기를 바꿔야 적절해 보입니다.

File: pytorch_vision_resnext.md [hunk 6]
```diff
@@ -52,51 +50,51 @@ preprocess = transforms.Compose([
...
     input_batch = input_batch.to('cuda')
     model.to('cuda')
 
 with torch.no_grad():
     output = model(input_batch)
-# Tensor of shape 1000, with confidence scores over Imagenet's 1000 classes
+# output은 shape가 [1000]인 Tensor 자료형이며, 이는 Imagenet 데이터셋의 1000개의 각 클래스에 대한 모델의 확신도(confidence)를 나타냅니다.
 print(output[0])
-# The output has unnormalized scores. To get probabilities, you can run a softmax on it.
+# output은 정규화되지 않았으므로, 확률화하기 위해 softmax 함수를 처리합니다.
```
Comment (Taeyoung96) [inline pytorch_vision_resnext.md:64]: 전체적으로 주석을 마무리할 때, 통일성을 가지면 좋을 것 같습니다.  
어떤 주석은 단어로 끝나고, 어떤 주석은 `~이다`로 끝나는 것 같네요. 😄

File: pytorch_vision_resnext.md [hunk 7]
```diff
@@ -25,23 +25,21 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'resnext50_32x4d', pretrained=T
...
-
-Here's a sample execution.
+모든 사전 훈련된 모델들은 입력 이미지가 동일한 방식으로 정규화되었다고 상정합니다.
+즉, 미니 배치(mini-batch)의 3-채널 RGB 이미지들은 `(3 x H x W)`의 형태를 가지며, 해당 `H`와 `W`는 최소 `224` 이상이어야 합니다.
+각 이미지는 `[0, 1]`의 범위 내에서 로드되어야 하며, `mean = [0.485, 0.456, 0.406]` 과 `std = [0.229, 0.224, 0.225]`을 이용해 정규화되어야 합니다.
+다음은 실행 예제 입니다.
 
 ```python
-# Download an example image from the pytorch website
+# 파이토치 웹 사이트에서 다운로드한 이미지 입니다.
```
Comment (Taeyoung96) [inline pytorch_vision_resnext.md:34]: 맞춤법 상 `이미지입니다.`로 붙여야 할 것 같습니다.

Comment (codingbowoo) [inline pytorch_vision_resnext.md:34]:
```suggestion
# 파이토치 웹 사이트에서 이미지 다운로드 
```
아래에서 create a mini-batch as expected by the model을 모델에서 가정하는대로 미니배치 생성 이라고 번역해주신 것처럼, 
이 부분도 주석을 보는 분들께 이부분은 다운로드 하는 부분이다~ 하는 내용이 전해져야 할 것 같습니다.

File: pytorch_vision_resnext.md [hunk 8]
```diff
@@ -52,51 +50,51 @@ preprocess = transforms.Compose([
...
 
-### Model Description
+### 모델 설명
 
-Resnext models were proposed in [Aggregated Residual Transformations for Deep Neural Networks](https://arxiv.org/abs/1611.05431).
-Here we have the 2 versions of resnet models, which contains 50, 101 layers repspectively.
-A comparison in model archetechure between resnet50 and resnext50 can be found in Table 1.
-Their 1-crop error rates on imagenet dataset with pretrained models are listed below.
+Resnext 모델은 논문 [Aggregated Residual Transformations for Deep Neural Networks]에서 제안되었습니다. (https://arxiv.org/abs/1611.05431).
+여기서는 50개의 레이어와 101개의 레이어를 가지는 2개의 resnet 모델을 제공하고 있습니다.
```
Comment (Taeyoung96) [inline pytorch_vision_resnext.md:89]: [용어 사용 규칙](https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md#%EC%9A%A9%EC%96%B4-%EC%82%AC%EC%9A%A9-%EA%B7%9C%EC%B9%99)을 참고하면 레이어는 계층으로 번역이 되는 것이 적절해 보입니다.

File: pytorch_vision_resnext.md [hunk 9]
```diff
@@ -52,51 +50,51 @@ preprocess = transforms.Compose([
...
-# move the input and model to GPU for speed if available
+# gpu를 사용할 수 있다면, 속도를 위해 입력과 모델을 gpu로 옮김
 if torch.cuda.is_available():
     input_batch = input_batch.to('cuda')
     model.to('cuda')
 
 with torch.no_grad():
     output = model(input_batch)
-# Tensor of shape 1000, with confidence scores over Imagenet's 1000 classes
+# output은 shape가 [1000]인 Tensor 자료형이며, 이는 Imagenet 데이터셋의 1000개의 각 클래스에 대한 모델의 확신도(confidence)를 나타냅니다.
```
Comment (Taeyoung96) [inline pytorch_vision_resnext.md:62]: Imagenet은 아래와 통일성을 갖기 위해 `ImageNet`으로 수정하는 것이 어떨까요?

**Review by Taeyoung96 — COMMENTED** [review]
@FollowerOfScriabin   
몇 가지 comment를 남겨봤습니다.  
확인해주시고 update해주시면 감사드리겠습니다.

File: pytorch_vision_resnext.md (hunk 5 above)
Comment (codingbowoo) [inline pytorch_vision_resnext.md:53]:
```suggestion
input_batch = input_tensor.unsqueeze(0) # 모델에서 가정하는 대로 미니배치 생성
```
https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=114792 에 따라 의존명사의 쓰임으로 보여, 띄어쓰기를 넣어주세요!

Comment (codingbowoo) [reply pytorch_vision_resnext.md:53]: https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=114792 에 따라 의존명사의 쓰임으로 보이네요.

File: pytorch_vision_resnext.md (hunk 3 above)
Comment (mindongjun) [reply pytorch_vision_resnext.md:28]: 안녕하세요 
해당 내용은 `가정`과 `상정`의 의미 차이에 따라
`상정합니다`가 더 적절한 뜻이 아닐까 생각됩니다.

> ‘상정’은 ‘딱 잘라서 판단하고 결정함’이라는 ‘단정’의 뜻이, ‘가정’은 ‘(임시로) 그렇다고 여김’이라는 ‘인정’의 뜻이 들어 있으므로, 이러한 뜻 차이를 고려하셔서 문맥에 맞게 ‘상정’이나 ‘가정’을 쓰시면 되겠습니다.
>
> 상정(想定)
>
> 「명사」
>
> 어떤 정황을 가정적으로 생각하여 단정함. 또는 그런 단정.
>
> ¶ 이 우스꽝스러운 상정을 남들이 일소에 부칠지라도 그때의 나로서는 꽤나 심각한 상정이었다.≪윤후명, 별보다 멀리≫
>
> 가정(假定)
>
> 「명사」
>
> 「1」사실이 아니거나 또는 사실인지 아닌지 분명하지 않은 것을 임시로 인정함.
>
> ¶ 그의 의식 속에는 만약이라는 가정이 항상 존재하고 있다./6월 초 선거가 실시된다는 가정 아래 준비를 해 왔다.

관련 국립국어원 내용 링크로 남겨드립니다.

https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=96465

**Comment by Taeyoung96** [comment]
@FollowerOfScriabin   
comment에 대해서 update 및 의견을 말씀해주시면 감사드리겠습니다.

**Review by Taeyoung96 — APPROVED** [review]
리뷰 사항 반영해주셔서 감사드립니다.

**Review by hyoyoung — APPROVED** [review]
good
